thread-manager/src/default_config.toml

```diff
@@ -0,0 +1,99 @@
+# pinned threads that do very important stuff
+[native_configs.solPohTickProd] # PoH thread
+max_threads = 1
+core_allocation.PinnedCores = { min = 0, max = 1 }
+
+[native_configs.solSigVerTpuVote]
+max_threads = 1
+
+[native_configs.solRcvrGossip] # Runs gossip receiver thread which seems to eat a lot of CPU all the time
+max_threads = 1
+core_allocation.PinnedCores = { min = 2, max = 3 }
+
+[native_configs.solSigVerTpu]
+max_threads = 1
+
+
+[native_configs.solClusterInfo]
+max_threads = 3
+
+
+[native_configs.solGossip]
+max_threads = 16
+
+[native_configs.solRepair]
+max_threads = 16
+
+[native_configs.FetchStage]
+max_threads = 16
+
+[native_configs.solShredFetch]
+max_threads = 16
+
+
+[rayon_configs.solReplayTx] #Replay transactions pool
+worker_threads = 12
+
+[rayon_configs.solReplayFork] # Replay forks pool
+worker_threads = 16
+
+[rayon_configs.solSvrfyShred] # runs turbine protocol signature verification for received shreds
+max_threads = 12
+
+[rayon_configs.solSigVerify] # Runs signature verification for cluster info
+max_threads = 8
+
+[rayon_configs.solRetransmit] # runs turbine protocol retransmit logic
+max_threads = 4
```

Comment:
I thought current Agave set this to 8

Reply:
Yes, these are all from optimized config. I have pulled them all from the PR for now and will initially use current values (even though these are "better").